examples/evpn/route-reflector/prepare.sh

```
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -euo pipefail
+set -x
+CURRENT_PATH=$(dirname "$0")
+
+source "${CURRENT_PATH}/../../common.sh"
+
+DEMO_MODE=true make deploy
+export KUBECONFIG=$(pwd)/bin/kubeconfig
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)